### ISSUE TITLE
Add support for custom log levels

### ISSUE THREAD

**Issue by alexeyqu** [uptrace/uptrace#376]
Hi, my project uses custom log levels which seem to be interpreted incorrectly by Uptrace.

For example, I am sending some logs with level `USAGE` which is between `INFO` and `WARN`.
In python that would be `(logging.INFO + logging.WARN)/2`.

`OTLPLogExporter` sends it like that (`severity_number` and `severity_text` fields):
```
    log_records {
      time_unix_nano: 1716187176431961088
      severity_number: SEVERITY_NUMBER_INFO4
      severity_text: "USAGE"
      body {
      . . .
```

However, Uptrace interprets it as `_system log:error`.
Is that something which can be fixed in Uptrace code? 

I tried looking over `pkg/tracing` folder but couldn't pinpoint the exact place where this conversion happens. 
Perhaps [here](https://github.com/uptrace/uptrace/blob/753aa3db89a2afb6def1d047b0409d9c1a2c5e03/pkg/tracing/span_attrs.go#L268-L287)?

Upd: seems that my guess is correct, I changed my log level name to `notice` (one of the aliases in linked code) like so:
```
    log_records {
      time_unix_nano: 1716188338725396480
      severity_number: SEVERITY_NUMBER_INFO4
      severity_text: "notice"
      body {
```
and it started to be shown as `INFO` on Uptrace.

Maybe it's better to categorize logs using `severity_number` OTLP constants ([reference](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitynumber))?

P.S. thanks for Uptrace, this is a neat tool.

**Comment by vmihailenco** [comment]
This should be fixed v1.7.5

**Comment by alexeyqu** [comment]
Wow. Thank you for such prompt response 🙏
